pulsar4s-avro/src/main/scala/com/sksamuel/pulsar4s/avro/package.scala

```diff
@@ -59,7 +60,7 @@ package object avro {
     }
 
     override def getSchemaInfo: SchemaInfo = {
-      SchemaInfo.builder()
+      SchemaInfoImpl.builder()
```

Comment:
Why the change to the impl ?

Reply:
`builder` is no longer a method of `SchemaInfo` in 2.8.0. That's the most natural replacement I could find after reading the source code of Apache Pulsar.

Reply:
Ok makes sense.